DirectEmployers/MyJobs
# [PD-2148] First stab at company access request form.

# Description
- User facing form to request access to a company. The company is *not* checked to exist (intentional)

# Known Issues
- ~~This thing is ugly. If you reference the jira ticket, you'll see that I'm awaiting feedback as to whether I should spend time sprucing it up, or if I should only be concerned with functionality~~ See PD-2151

# Testing
1. Visit /request-company-access
2. Enter a company name
3. Note that you get an access code

There is a test that ensures that that code is valid, but if you'd like to prove it to yourself, you can do something like this in a console:
```
>>> from myjobs.models import CompanyAccessRequest
>>> a = CompanyAccessRequest.objects.last()
>>> print a.check_access_code("<code from form here>")
```
If the above returns False, we have a problem. Otherwise, we are good to go.
